### PR TITLE
basic world file

### PR DESCRIPTION
**Pull request by isro01**
This has the .world file and the model.sdf file in the iros2016_track folder

### CODE REVIEW

**Review by rohanblueboybaijal — APPROVED** [review]

**Review by ashwin2802 — CHANGES_REQUESTED** [review]
rename the basic_world to worlds. We will put this in one of the packages later. i think iros2016_track should be moved to a directory specifically containing external submodules (@gajena). don't move that for now.

**Review by ashwin2802 — APPROVED** [review]

**Review by PenceMataria — APPROVED** [review]

**Review by PenceMataria — COMMENTED** [review]
Can have a proper repo with all our gazebo customization/creations/borrowings which can be linked. Might be useful for later stages too.

File: simulator/package.xml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+<package format="2">
+  <name>simulator</name>
+  <version>0.0.0</version>
+  <description>The simulator package</description>
+
+  <maintainer email="isro@todo.todo">isro</maintainer>
+
+  <license>TODO</license>
+
+  <!-- The *depend tags are used to specify dependencies -->
```
Comment (gajena) [inline simulator/package.xml]: remove these comments.

File: simulator/worlds/blue_frames.world [hunk 2]
```diff
@@ -0,0 +1,248 @@
+<sdf version='1.6'>
```
Comment (gajena) [inline simulator/worlds/blue_frames.world]: name of the file must be intutive, like this is world file then name something like basic_arena ... something something

**Comment by ashwin2802** [comment]
great work!

**Pull request merged**